muse/components/load_balancer.py

```diff
@@ -258,9 +260,20 @@ def update_servers(self, server_works: List[L.LightningWork]):
             print("deleted servers:", deleted_servers)
 
         servers = self.servers
+        AUTHORIZATION_TYPE = "Basic"
```

Comment:
why as "constant" when it is used only once?

Reply:
i think it's okay, it's a value that won't be changed. hence a constant